### PR TITLE
Fix minor issues in UG for cs2101

### PR DESCRIPTION
**Pull request by nananakx-x**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ title: User Guide
 Here are some tasks which Foster Family can help you with: 
 * **Store and update** the important details of animal fosterers (people who temporarily care for an animal in their own homes).
 * **Search** for a fosterer using _any_ detail you can remember of them.
-* **Gain insights** on the overall status of managed fosterers.
+* **Gain insights** on the overall status of fosterers managed.
```
Comment (butteredyakiimo) [inline docs/UserGuide.md]: actually! i feel like managed fosterers flows better for me :O wdyt!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -296,17 +295,17 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS housing/HOUSING_TYPE availa
...
+| `PHONE_NUMBER`   | Phone number of the fosterer                                                                                                                                      | `93456778`, `89067547`                                      |
+| `EMAIL`          | Email of the fosterer                                                                                                                                             | `thomas718@gmail.com`, `kate@yahoo.com.sg`                  |
+| `ADDRESS`        | Address of the fosterer                                                                                                                                           | `Orchard road, Blk 8, #13-04`                               |
+| `HOUSING_TYPE`   | - Housing type of the fosterer<br/> - Case-sensitive<br/> - Can only take the values shown in the example column                                                  | `HDB`, `Condo`, `Landed`, `nil`                             |
```
Comment (butteredyakiimo) [inline docs/UserGuide.md:304]: ye ! i think this works :D

**Review by butteredyakiimo — DISMISSED** [review]
very minor, can ignore if not needed!

File: docs/UserGuide.md (hunk 1 above)
Comment (nananakx-x) [reply docs/UserGuide.md]: ok wait can HAHAHA i read it out it seems ok also

**Review by butteredyakiimo — APPROVED** [review]

**Pull request merged**